utils/wm-utils/src/main/java/com/microsoft/device/dualscreen/utils/wm/WindowLayoutInfoExtensions.kt

```diff
@@ -24,6 +24,14 @@ fun WindowLayoutInfo?.isFoldingFeatureVertical(): Boolean =
     this != null &&
         (displayFeatures.firstOrNull() as? FoldingFeature)?.orientation == FoldingFeature.Orientation.VERTICAL
 
+/**
+ * Checks whether the orientation of the folding feature is horizontal
+ * @return true if the folding feature exists and is horizontal oriented, false otherwise
+ */
+fun WindowLayoutInfo?.isFoldingFeatureHorizontal(): Boolean =
```

Comment:
Please use WM `isSeparating` instead this custom function.

Reply:
isSeparating is used to see if the entire window should is split into multiple physical areas.
We can have the case when the isSeparating is true, but the NavigationRailView does not intersect the folding feature and thus should not split it's content.